test/unit-tests/expression/security.test.js

```diff
@@ -32,11 +32,11 @@ describe('security', function () {
 
   it('should not allow calling constructor', function () {
     assert.throws(function () {
-      math.evaluate('constructor')
+      math.evaluate('constructor', {})
     }, /Error: No access to property "constructor"/)
 
     assert.throws(function () {
-      math.evaluate('toString')
+      math.evaluate('toString', {})
```

Comment:
These shouldn't need changing, because `validateScope` is still checking against the `keywords` prototype.